Makefile

```diff
@@ -326,14 +326,14 @@ $(foreach ITEM,$(PILOT_GO_BINS_SHORT),$(eval $(call pilotbuild,$(ITEM))))
 .PHONY: istioctl
 istioctl ${ISTIO_OUT}/istioctl:
 	bin/gobuild.sh ${ISTIO_OUT}/istioctl ./istioctl/cmd/istioctl
-
+    
 # Non-static istioctls. These are typically a build artifact.
 ${ISTIO_OUT}/istioctl-linux: depend
-	STATIC=0 GOOS=linux   bin/gobuild.sh $@ ./istioctl/cmd/istioctl
+	STATIC=0 GOOS=linux   bin/gobuild.sh -s -w $@ ./istioctl/cmd/istioctl
```

Comment:
whats `-s -w`?

Reply:
Is this going to work?  I modified the line `LDFLAGS="-extldflags -static"` in _gobuild.sh_ by adding -s and -w and built _istioctl_.  This did what I wanted: reduced the size by 10% and the line numbers were still present on the backtrace.  (See https://blog.filippo.io/shrink-your-go-binaries-with-this-one-weird-trick/ ).  You need to test with `make istioctl` and verify the resulting binary is smaller, and also compile the other Istio binaries and verify they are unchanged.